CONTRIBUTING.md

```diff
@@ -197,6 +201,16 @@ If that gets executed in one of the package folders it will only run the tests
 of this package. If it gets executed in the n8n-root folder it will run all
 tests of all packages.
 
+#### E2E tests
+
+E2E tests can be started via one of the following commands:
+
+- `pnpm test:e2e:ui`: Run e2e tests interactively against port 5678
+- `pnpm test:e2e:dev`: Run e2e tests interactively against port 8080 (editor-ui dev server, needs to be running separately)
```

Comment:
The command starts a dev server automatically so you don't have to run it separately.
Maybe what's important here is it reacts to editor-ui code changes.